llvm/test/CodeGen/X86/section-stats.ll

```diff
@@ -0,0 +1,10 @@
+; REQUIRES: asserts
+; RUN: llc -o /dev/null -filetype=obj -stats %s 2>&1 | FileCheck %s
+
+; CHECK: {{[0-9+]}} elf-object-writer - Total size of SHF_ALLOC text sections
```

Comment:
`[[#]]`. I think it's fine to test the exact number to ensure that we don't regress.

You can add a variable and test SHF_WRITE as well.

Reply:
done